samples/CoreAnimationTest/CoreAnimationTest.vsimporter/CoreAnimationTest-WinStore10/CoreAnimationTest-Debug-xcvars.txt

```diff
@@ -87,15 +87,15 @@ PRODUCT_BUNDLE_IDENTIFIER = com.Microsoft.CoreAnimationTest
 PRODUCT_NAME = CoreAnimationTest
 PRODUCT_TYPE = com.apple.product-type.application
 PROJECT = CoreAnimationTest
-PROJECT_DIR = E:/GIT_REPOS/winobjc/samples/CoreAnimationTest
-PROJECT_FILE_PATH = E:\GIT_REPOS\winobjc\samples\CoreAnimationTest\CoreAnimationTest.xcodeproj
+PROJECT_DIR = Y:/Arcadia/WinObjC/samples/CoreAnimationTest
+PROJECT_FILE_PATH = Y:\Arcadia\WinObjC\samples\CoreAnimationTest\CoreAnimationTest.xcodeproj
```

Comment:
I thought this file was never used.  My *limited* understanding is that it was just output.  Could be wrong.

---
In reply to: [102826848](https://github.com/Microsoft/WinObjC/pull/2056#discussion_r102826848) [](ancestors = 102826848,102809102)